vagrant/ubuntu15.04/Vagrantfile

```diff
@@ -0,0 +1,66 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+####################################
+#       Ubutntu 15.04 Build
+#
+# Downloads and installs the latest
+# Ubuntu 12.04 build posted to S3 
+# from Jenkins in a Ubuntu15.04 VM.
+#
+# IP: 10.0.0.88
+#
+####################################
+
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # All Vagrant configuration is done here. The most common configuration
+  # options are documented and commented below. For a complete reference,
+  # please see the online documentation at vagrantup.com.
+
+  # Every Vagrant virtual environment requires a box to build off of.
+  config.vm.box = "ubuntu/vivid64"
```

Comment:
Apparently not. It seems to automatically get pulled from Atlas now, much like the docker hub or whatever.